code/game/objects/effects/aliens.dm

```diff
@@ -479,9 +479,18 @@
 			T.dump_contents()
 			qdel(target)
 
+		if(istype(target, /turf/simulated/mineral))
+			var/turf/simulated/mineral/M = target
+			M.ChangeTurf(M.baseturf)
+
+		if(istype(target, /turf/simulated/floor))
+			var/turf/simulated/floor/F = target
+			F.ChangeTurf(F.baseturf)
+

```

Comment:
Unless I am remembering it wrong, when I tried that, it gave a compile error. I might be wrong though, but it does work like this regardless.